tasks/sendgridsync.js

```diff
@@ -0,0 +1,45 @@
+const sgClient = require("@sendgrid/client");
+
+const sendgridSync = async () => {
+  //Lazy Load Models so you can change process.ENV dynamically
+  const models = require("../api/models");
+  sgClient.setApiKey(process.env.SENDGRID_API_KEY);
+
+  const hackerProfiles = await models.HackerProfile.findAll({});
+
+  // Applicants Odyssey List in Sendgrid
+  const HackersListID = "95f43e1d-83c2-4818-a7d7-19eb44c30ae4";
```

Comment:
Is it gonna be a problem that this is checked into source control?

Reply:
It's less than ideal but fine non-secure item

Reply:
If we ever have to change it, will move to an ENV. It's not a security risk just a magic string

Reply:
If we ever have to change it, will move to an ENV. It's not a security risk just a magic string

Reply:
Okay, yeah that's fine just wanted to make sure this wasn't supposed to be secret